src/gmp.c

```diff
@@ -10545,9 +10543,7 @@ results_xml_append_nvt (iterator_t *results, GString *buffer, int cert_loaded)
                                     "<family/>"
                                     "<cvss_base>%s</cvss_base>"
                                     "<cve>%s</cve>"
```

Comment:
Initially, yes. Because it was the nvt type "ovaldef".
But in fact it is a nvt element and so it should better follow the same scheme.
I added a commit to solve it.

Reply:
Great, thanks.